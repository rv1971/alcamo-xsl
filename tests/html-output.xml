<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc
    xmlns:dc="http://purl.org/dc/terms/"
    xmlns:owl="http://www.w3.org/2002/07/owl#"
    xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#"
    dc:identifier="html-output"
    dc:title="Test for html output"
    dc:creator="Alice"
    dc:publisher="example.org"
    dc:created="2023-04-18T11:57:00+02:00"
    dc:modified="2023-04-18T11:57:05+02:00"
    dc:accessRights="public"
    dc:rights="Â© Example.org 2023"
    dc:rigtsHolder="Example.org"
    owl:versionInfo="0.42.0">
  <xsl:stylesheet
      xmlns="http://www.w3.org/1999/xhtml"
      xmlns:xh="http://www.w3.org/1999/xhtml"
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#"
      xml:id="xslt"
      version="1.0"
      exclude-result-prefixes="a xh">
    <xsl:import href="../xsl/html-output.xsl"/>

    <xsl:param name="a:cssList" select="'foo.css  bar.css baz.css'"/>
    <xsl:param name="a:jsList" select="'qux.js foo.js'"/>

    <xsl:template match="/">
      <html xmlns:dc="http://purl.org/dc/terms/">
        <xsl:call-template name="a:head"/>
      </html>
    </xsl:template>
  </xsl:stylesheet>
</a:doc>
