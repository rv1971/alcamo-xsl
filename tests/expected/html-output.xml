<?xml version="1.0" encoding="UTF-8"?>
<html xmlns:dc="http://purl.org/dc/terms/" xmlns="http://www.w3.org/1999/xhtml">
  <head xmlns:owl="http://www.w3.org/2002/07/owl#">
    <meta charset="utf-8"/>
    <meta property="dc:identifier" content="html-output"/>
    <title property="dc:title">Test for html output</title>
    <meta property="dc:creator" name="author" content="Alice"/>
    <meta property="dc:created" content="2023-04-18T11:57:00+02:00"/>
    <meta property="dc:modified" content="2023-04-18T11:57:05+02:00"/>
    <meta property="dc:accessRights" content="public"/>
    <meta property="dc:rights" content="Â© Example.org 2023"/>
    <meta property="owl:versionInfo" content="0.42.0"/>
    <link rel="stylesheet" type="text/css" href="foo.css"/>
    <link rel="stylesheet" type="text/css" href="bar.css"/>
    <link rel="stylesheet" type="text/css" href="baz.css"/>
    <script type="text/javascript" src="qux.js"> </script>
    <script type="text/javascript" src="foo.js"> </script>
  </head>
  <body>
    <p id="p1">Paragraph 1</p>
    <p>xslt</p>
    <p>p1</p>
    <a href="#p1">Paragraph 1</a>
  </body>
</html>
